apps/percona/chaos/cstor_pool_scale/test.yml

```diff
@@ -0,0 +1,179 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+    - block:
+
+        ## RECORD START-OF-TEST IN LITMUS RESULT CR
+
+        - block:
+
+            - name: Record test instance/run ID
+              set_fact:
+                run_id: "{{ lookup('env','RUN_ID') }}"
+
+            - name: Construct testname appended with runID
+              set_fact:
+                test_name: "{{ test_name }}-{{ run_id }}"
+
+          when: lookup('env','RUN_ID')
```

Comment:
@shashank855  Can you use this util `/common/utils/create_testname.yml` to create test name.

Reply:
@nsathyaseelan I have made the changes !!